docs/docs/gatsby-core-philosophy.md

```diff
@@ -1,168 +1,168 @@
 ---
-title: Gatsby Core Philosophy
+title: Filosofia central do Gatsby
 ---
 
-Gatsby's core philosophy can be divided into three parts.
+A filosofia central de Gatsby pode ser dividida em três partes.
 
-**First, [our vision](#the-gatsby-vision).** Our vision is to _[construct higher-level web building blocks](#construct-new-higher-level-web-building-blocks)_, and _[create a cohesive content mesh system](#create-a-cohesive-content-mesh-system)_, in order to _[make building websites fun](#make-building-websites-fun)_ and _[build a better web](#build-a-better-web)_.
+**Primeiro, [nossa visão](#a-visão-do-gatsby).** Nossa visão é _[construir novos alicerces da Web de alto nível](#construir-novos-alicerces-da-web-de-alto-nível)_, e _[criar um sistema de malha de conteúdo coeso](#crie-um-sistema-de-malha-de-conteúdo-coeso)_, para _[tornar a construção de sites divertida](#tornar-a-construção-de-sites-divertida)_ e _[criar uma web melhor](#crie-uma-web-melhor)_.
```

Comment:
```suggestion
**Primeiro, [nossa visão](#a-visão-do-gatsby).** Nossa visão é _[construir novos alicerces da Web de alto nível](#construir-novos-alicerces-da-web-de-alto-nível)_, e _[criar um sistema de malha de conteúdo coeso](#crie-um-sistema-de-malha-de-conteúdo-coeso)_, para _[tornar a construção de sites divertida](#tornar-a-construção-de-sites-divertida)_ e _[criar uma web melhor](#crie-uma-web-melhor)_.
```

Reply:
A tradução dentro do link mantém a funcionalidade de acordo? [nossa visão](#**a-visão-do-gatsby**), nesse caso seria um link para https://www.gatsbyjs.org/docs/gatsby-core-philosophy/#**the-gatsby-vision**, o id do H2 é gerado automaticamente depois?

Reply:
Olha no [meu fork](https://github.com/DouglasMV/gatsby-pt-BR/blob/traducao-gatsby-core-philosophy/docs/docs/gatsby-core-philosophy.md). Todos links internos (para dentro do próprio documento) funcionam.

Alguns links para outras páginas não funcionam (linhas 30, 38, 64, 90, 114, 158, 159, 164) porém eles também não funcionam no [repositório raiz](https://github.com/gatsbyjs/gatsby-pt-BR/blob/master/docs/docs/gatsby-core-philosophy.md)